apiserver/application/application_test.go

```diff
@@ -2556,46 +2567,117 @@ func remoteOffers() []params.ApplicationOffer {
 
 func (s *serviceSuite) TestSuccessfullyAddRemoteRelation(c *gc.C) {
 	s.offersApiFactory.offers = remoteOffers()
-	endpoints := []string{"wordpress", "local:/u/me/prod/hosted-mysql"}
+	endpoints := []string{"wordpress", "local:/u/me/hosted-mysql"}
+	s.assertAddRelation(c, endpoints)
+}
+
+func (s *serviceSuite) addTestingCharmOtherModel(c *gc.C, name string) *state.Charm {
```

Comment:
This helper method operates on the test suite attribute "otherModel", which is created in test setup. So OtherModel in the method title refers to that attribute name.